doc/kubernetes/modules/ROOT/partials/isup-bash-version-macos.adoc

```diff
@@ -0,0 +1,9 @@
+[WARNING]
+====
+The script `isup.sh` currently works with macOS  on bash version 4 or higher. Current Bash version can be checked using the command:
```

Comment:
@nehachopra27 thanks for the suggestion, its great, the script would work with just a bash version upgrade. Can you refer to any docs which could help with the upgrade and highlight the fact that zsh is the default shell now thats bundled with the MacOS ?